### DISCUSSION TITLE
Replacement for Wizard's lesser forcewall

### DISCUSSION THREAD

**Discussion by Contrabang** [ParadiseSS13/Paradise#19266] [Documents]
## Problem
Wizard's 1x1 forcewall spell is basically just noobbait. 3x1 forcewall is better in it than every way but cooldown, making almost no one every using lesser forcewall. The only place where this spell can be used is in doorways or maints, where wizards typically need to use it in the main halls, where it would do nothing against the horde of the tide.

The only thing unique about it, is that it doesnt require wizard clothes. But, you can just get around that for the current one with the no clothes spell, and once you use the lesser forcewall spell, you're exposed as a wizard anyways, not really worth it.

## A solution?
Replace lesser forcewall spell in the spell book with the current greater one, then create a new greater one. For this example, it would be a 2 (or 3?) spell cost spell, with a fairly high cooldown, perhaps 1 spell per 30 seconds. It would create 6 tiles of forcewalls, 3 on either side of the user. This puts them in a "half-box", surrounded on all sides except two, the users front and back. Now, this can be used skillfully by turning before using the spell to create a small box to protect themselves, if they place it in the right way. Combining the two new forcewalls would give the user the ability to create a 3x3 spot of entire forcewalls, allowing them to either give themselves full protection or allowing them to trap a target and assassinate them. 

This method of changing this would not change the mime's forcewall or traitor forcewall items.

This would leave the current lesser forcewall for any wizard spell groups (like the greytider I think has it?) as its current use.

**Comment by AzuleUtama** [comment]
Honestly, I'd just merge the normal and greater forcewall into a single spell and be done with it, the 1x1 wall is useless and as you say, noob bait for newbie wizards who don't know the difference. Having two versions of the spell is just more confusing if anything. The only real consideration would be that people who get forcewall through single-use spellbooks and healer apprentices would be buffed but I doubt that's a massive concern given how rare both are.

I'd argue the same for greater knock as well, the "unlock all airlocks" feature should just be its own "summon" you can purchase for 1SP and knock should just be its own spell you can buy alongside or without it.

**Reply by hal9000PR** [reply]
This solution sounds good. OR make the "new greater forcewall" act similarly to hemomancer blood barrier.
